torch/testing/_internal/common_methods_invocations.py

```diff
@@ -17628,6 +17628,90 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
         dtypesIfCUDA=floating_types_and(torch.float16, torch.bfloat16),
         sample_inputs_func=sample_inputs_scatter_reduce,
     ),
+    BinaryUfuncInfo(
+        'special.jacobi_elliptic_k_cd',
+        dtypes=all_types_and(torch.bool),
```

Comment:
FYI in case you didn't find it, this is implemented here:

https://github.com/pytorch/pytorch/blob/97deda4f281e18b66d0e841a15d424bd6701d374/torch/testing/_legacy.py#L89

These names are kinda wonky, but they are consistent with the C++ dispatch macros. I think TensorIterator is autopromoting integer inputs to floating input for these functions, which is why the macros are divergent here and in the code above (where they just apply to "FLOATING_TYPES"). Because the macros aren't updated as we add new types, FLOATING_TYPES is float and double, and ALL_TYPES is float and double and torch.uint8, torch.int8, torch.int16, torch.int32, and torch.int64